lua/libs/settings.lua

```diff
@@ -9,19 +9,36 @@ DBPASS = 'password'
 DBPORT = 5432
 
 --
--- Select the TTS engine, value : flite, acapela
+-- Select the TTS engine, value : flite, acapela, msspeak
 --
 TTS_ENGINE = 'flite'
 
 --
--- Acapela TTS Settings
+-- Acapela and MSSpeak TTS Settings
+-- MSSpeak only needs APPLICATION_LOGIN(client_id)
+-- and APPLICATION_PASSWORD(client_secret).
+-- ACCOUNT_LOGIN is ignored for MSSpeak
 --
 ACCOUNT_LOGIN = 'EVAL_VAAS'
 APPLICATION_LOGIN = 'EVAL_YYYYYYY'
 APPLICATION_PASSWORD = 'XXXXXXXX'
 
+--
+-- For MSSpeak comment out Acapela SERVICE_URL
+-- and uncomment other SERVICE_URL parameter
+--
+-- SERVICE_URL = 'http://api.microsofttranslator.com/V2/Http.svc/Speak'
```

Comment:
let's try to avoir service_url for MSTranslate, and use the default value, this is likely not to change for a long time